pkg/cli/cli.go

```diff
@@ -57,5 +64,8 @@ A very simple cli.`,
 		newDeleteCmd(client),
 	)
 
+	rootCmd.PersistentFlags().IntVarP(&cfg.LogLevel, "verbosity", "v", 0, "level of log verbosity")
+	rootCmd.PersistentFlags().StringVarP(&cfg.LogFileName, "log-file", "l", "", "print logs to file")
```

Comment:
`l` is a bit "obscure", I would just use the `log-file` flag.

```suggestion
	rootCmd.PersistentFlags().StringVar(&cfg.LogFileName, "log-file", "", "print logs to file")
```